Automattic/WP-Job-Manager
# Fix the input for the datepicker to always submit yy-m-d

Fixes issue from #1315 

#### Changes proposed in this Pull Request:

* `altFormat` doesn't do anything unless an `altField` is provided in jQuery UI's `datepicker`. This creates a no-JS safe solution to insert a hidden field that actually has the submitted value in the correct format.

#### Testing instructions:

* In WP Admin, edit the expiry date and verify it always saves in database correctly as yy-m-d no matter what the date format setting is for WP core.
